docker/Dockerfile

```diff
@@ -24,12 +33,11 @@ RUN sh /build/install_build_deps.sh
 COPY go.mod /build
 WORKDIR /build
 RUN go mod download
-
 COPY . /build
+#RUN --mount=type=cache,target=/root/.cache/go-build rm -f server/service && make bins
+RUN rm -f server/service && make bins
```

Comment:
mounting cache here, significantly speeds up the build

Reply:
Thanks for the feedback, make sense, assumed so! I'm trying to figure out what's braking build in that it cannot find the protoc helper and tried to disable the BUILDKIT which helped me to get around the error locally. Unfortunately, as we can see it did not solve the build issue here so I'll be reverting this change soon.

Reply:
I added it back, didn't really help here.